examples/server-side-rendering/package.json

```diff
@@ -29,25 +29,25 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^5.0.0",
-    "@angular/common": "^5.0.0",
-    "@angular/compiler": "^5.0.0",
-    "@angular/core": "^5.0.0",
-    "@angular/forms": "^5.0.0",
-    "@angular/http": "^5.0.0",
-    "@angular/platform-browser": "^5.0.0",
-    "@angular/platform-browser-dynamic": "^5.0.0",
-    "@angular/platform-server": "^5.0.0",
-    "@angular/router": "^5.0.0",
-    "@nguniversal/express-engine": "^5.0.0-beta.5",
-    "@nguniversal/module-map-ngfactory-loader": "^5.0.0-beta.5",
-    "angular-instantsearch": "1.0.1",
-    "core-js": "^2.4.1",
-    "instantsearch.js": "^2.3.0",
-    "nouislider": "^10.1.0",
-    "preboot": "^6.0.0-beta.0",
-    "rxjs": "^5.5.2",
-    "zone.js": "^0.8.14"
+    "@angular/animations": "5.0.0",
```

Comment:
Worth trying update to v6

Reply:
It [seems migration from 5 to 6 is not straight forward](https://stackoverflow.com/questions/48970553/want-to-upgrade-project-from-angular-v5-to-angular-v6#answer-49474334). There are a few changes required to the code. I'll isolate them in a another PR if you don't mind